src/styles/_services.scss

```diff
@@ -2,7 +2,119 @@
   #hero {
     height: 100vh;
     min-height: 700px;
-    background-image: url("/assets/img/services/finsecptx-service-banner-2000.jpg");
+    background-image: url("/assets/img/services/hero-phone-boot-bg-2000.jpg");
+    background-position: center;
+    background-size: cover;
```

Comment:
If you want, you can use shorthands like `background: url(' ... ') center / cover` to achieve the same in one line. Here is a bit more info: http://www.w3schools.com/cssref/css3_pr_background.asp